packages/top-nav/stories/top-nav.stories.ts

```diff
@@ -106,3 +107,44 @@ export const Selected = (): TemplateResult => {
         </sp-top-nav>
     `;
 };
+
+// https://spectrum.adobe.com/page/application-frame/#Application-mode
+// https://spectrum.adobe.com/page/headers/#Anatomy
+
+export const Modes = (): TemplateResult => {
+    return html`
+        <sp-top-nav>
+            <sp-action-button quiet href="/marketing-page" label="App">
+                <sp-icon slot="icon" src=${logo}></sp-icon>
+            </sp-action-button>
+            <sp-top-nav-item href="#">Design</sp-top-nav-item>
+            <sp-top-nav-item href='#prototype'>Prototype</sp-top-nav-item>
+            <span style="margin-inline-start: auto; font-size: 1.1em;">document_name.pdf</span>
+            <sp-button-group style="margin-inline-start: auto;">
```

Comment:
Please make this a `sp-action-group` element instead. It has styling (and will have functionality in an upcoming release) specific to leveraging `sp-action-buttons`.